charts/infra-apps/values.yaml

```diff
@@ -0,0 +1,118 @@
+namespaces:
+  # namespaces.enabled -- Enable creating infra Namespaces
+  enabled: true
+  # namespaces.namespaces -- Map of Namespaces
+  # @default -- see [values.yaml](./values.yaml)
+  namespaces:
+    infra-argocd:
+      enabled: true
```

Comment:
I don't like that this enables to many workspaces and isn't coupled to the namespaces used below.

Maybe we get rid of namespaces here with argo events/workflow or this needs refactoring once we have a library chart.